packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -93,9 +93,10 @@ export default {
             modalContext.open();
         }
 
-        if (this.isDuplicateOrderError) {
-            this.trackDuplicateOrderWarnDialog();
-        }
+        this.trackDialogEvent({
+            code: this.message?.code,
+            isDuplicateOrderError: this.isDuplicateOrderError
```

Comment:
If `isDuplicateOrderError` is `false`, why would we log an event at all?

Reply:
Because the dialog will still be showing an error, just not a duplicate order error